frontend/package.json

```diff
@@ -18,13 +18,16 @@
     "devDependencies": {
         "@types/react": "18.3.3",
         "@types/react-dom": "18.3.0",
+        "@vite-pwa/assets-generator": "0.2.4",
         "@vitejs/plugin-react": "4.3.1",
         "autoprefixer": "10.4.20",
         "eslint-plugin-jsx-a11y": "6.9.0",
         "eslint-plugin-react": "7.35.0",
         "eslint-plugin-react-hooks": "4.6.2",
         "postcss": "8.4.41",
-        "vite": "5.4.0"
+        "vite": "5.4.0",
+        "vite-plugin-pwa": "0.20.1",
+        "workbox-core": "7.1.0"
```

Comment:
why we need this package?

Reply:
From what I understand, the Vite PWA plugin will use the Workbox Core library to interact with and build the service worker.
https://vite-pwa-org.netlify.app/workbox/